README.md

```diff
@@ -37,79 +30,136 @@ Flyte is an **open-source orchestrator** that facilitates building production-gr
     <img src="https://img.shields.io/badge/Slack-Chat-pink?style=for-the-badge&logo=slack" alt="Flyte Slack label" /></a>
 </p>
 
+Flyte is an open-source orchestrator that facilitates building production-grade data and ML pipelines. It is built for scalability and reproducibility, leveraging Kubernetes as its underlying platform. With Flyte, user teams can construct pipelines using the Python SDK, and seamlessly deploy them on both cloud and on-premises environments, enabling distributed processing and efficient resource utilization.
+
+<h2 align="center">
+  Build
+</h2>
+<p>
+Write code in Python or any other language and leverage a robust type engine. 
+</p>
+
+<img alt="Getting started with Flyte" src="https://raw.githubusercontent.com/flyteorg/static-resources/main/common/flytereadmebuildv2.gif" style="width: 100%; height: auto;" />
```

Comment:
can we reduce the size of the GIF window?

Reply:
also, i'm not sure if this is a big ask, but could we consider including a different example here? :)
[this](https://docs.flyte.org/projects/cookbook/en/latest/auto_examples/basics/workflow.html) would be amazing.

Reply:
That's a good example yes, the problem is that the GIF will be a bit longer (more code) and this one doesn't showcase the subworkfows that are probably not the most prominent feature but something a bit beyond the hello world